deliverables/loama/design.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Design
+---
+# Design
+
+In order to guide the development and gain early feedback on the product, we've created a Hi-fi prototype of the application that can can be used for testing.
+
+This proototype is composed out of 4 parts:
+- The login & sign up flow
+- The file explorer section
+- The permissions sidebar
+- The 4-step access assignment
+
+## Design Principles
+The design for LOAMA is built upon a set of fundamental design principles:
+
+- **Visibility of System Status**: In order to help the user orientate themselves throughout the application, we've made sure that the user **knows** in which part of the app they are. Some examples of this are:
+    - The hightlight menu tab
+    - The left navigation bar on the file section
+- **Error Prevention & Recovery**: Because Solid is somewhat difficult to understand at certain times, we want to remove any files or settings that can put the user at risk and help them recover from any error prevention. Some examples are:
+    - The warning indication for dangerous permissions
+    - The warning dialog box and the button to easily remove the mentioned permission
+- **Help & Documentation**: People aren't familiar with some Solid terminology. If we can't replace the solid term with a more well-known alternative, we explain that term to the user. This is done by placing tooltips with every term that contain an explanation of the term.
+
+## Design System
+
+Our in-house design system is based upon the color palette & typography chosen in the [branding section](/project/brand.md). For the project, we've created multiple components with the appropriate states *(Like error states, etc...)*
+
+## Prototype
+
+The figma design that you see below here is fully prototyped and can be explored. You can use the interface below of visit the file [here](https://www.figma.com/proto/Db518nvwhA0tiQQa47e7pr/AMA-Prototype?node-id=97-943&t=mG2chueVNFUDZRNz-1&scaling=scale-down-width&content-scaling=fixed&page-id=97%3A2&starting-point-node-id=97%3A943).
+
+<iframe style="border: 1px solid rgba(0, 0, 0, 0.1);" width="800" height="800" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Fproto%2FDb518nvwhA0tiQQa47e7pr%2FAMA-Prototype%3Fnode-id%3D97-943%26t%3DmG2chueVNFUDZRNz-1%26scaling%3Dscale-down-width%26content-scaling%3Dfixed%26page-id%3D97%253A2%26starting-point-node-id%3D97%253A943" allowfullscreen></iframe>
+
+## User testing
+
+During the demo day, we've done a wider user test in order to gain more insight on certain flows and if the design could be improved. The report for this can be seen below or access by [clicking here](https://app.maze.co/report/Loama-H-V1/5be3k7lyzw1jys/intro).
```

Comment:
```suggestion
During the demo day, we've done a wider user test in order to gain more insight on certain flows and if the design could be improved. The report for this can be seen below or accessed by [clicking here](https://app.maze.co/report/Loama-H-V1/5be3k7lyzw1jys/intro).
```